test/integration/user_authorization_test.rb

```diff
@@ -6,10 +6,28 @@ class UserAuthorizationTest < ActionDispatch::IntegrationTest
 
   def setup
     @user = User.create(username: 'user',
-    password: 'password',
-    first_name: 'John',
-    last_name: 'Doe',
-    email: 'example@example.com')
-    visit root_url
+                        password: 'password',
+                        first_name: 'John',
+                        last_name: 'Doe',
+                        email: 'example@example.com',
+                        role: 1)
+                        visit root_url
+  end
+
+  test "an unauthorized user cannot see another user's order history" do
+    user1 = User.create(username: 'user',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.